.gitbook.yaml

```diff
@@ -0,0 +1,8 @@
+root: ./
+
+structure:  
+   summary: SUMMARY.md
+    
+redirects:  
+    /company/how-to-guides-for-staff/how-to-spend-company-money: /operations/finance/how-to-spend-company-money/README.md
+    /company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract: /operations/finance/spending-company-money/how-to-procure-a-vendor-contract.md
```

Comment:
```suggestion
    /company/how-to-guides-for-staff/how-to-purchase/how-to-procure-a-vendor-contract: /operations/finance/spending-company-money/procurement/how-to-procure-a-vendor-contract.md
```

Reply:
I'm not entirely sure, but looks like the procure vendor contract is now under procurement?

Reply:
This is what Tim suggested in the spreadsheet and I *think* that's how it should be presented.

<img width="1567" alt="Screen Shot 2020-10-23 at 14 47 25" src="https://user-images.githubusercontent.com/4223530/97005389-ad9f8380-153e-11eb-8f2d-206dc5f8a696.png">

Reply:
@justinegeffen in the SUMMARY file it's under procurement

![image](https://user-images.githubusercontent.com/13119842/97014694-aa919c80-1518-11eb-970b-900dde21f93b.png)

Reply:
@justinegeffen How you have the yaml file set up might be fine, I'm not sure how the redirect logic works.

Reply:
I'm not either - I'd like to test what I have to figure out whether I've understood it properly. I was considering adding all the redirects and testing them all but if I did misunderstand the structure then I don't want to break it all. Thoughts on merging to test?

Reply:
Yea, I'm good to test this out, approving @justinegeffen